src/main/java/service/read/impl/ProcessFileContentImpl.java

```diff
@@ -0,0 +1,49 @@
+package service.read.impl;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import service.read.OperationTypeService;
+import service.read.ProcessFileContent;
+
+public class ProcessFileContentImpl implements ProcessFileContent {
+    private static final String CSV_SEPARATOR = ",";
+    private static final String OPERATION_TYPE_NAME = "type";
+    private static final String FRUIT_QUANTITY_NAME = "quantity";
+    private static final String FRUIT_NAME = "fruit";
```

Comment:
to make our solution more flexible, we can get this values through constructor,
so we will be able to use `,` or `;` as a separator and another names for columns